index.html

```diff
@@ -212,16 +250,14 @@ <h1 class="contact_me">Contact me</h1>
    <div>
    <textarea class="textarea_form" required rows="4" maxlength="500" placeholder="Write your message here" cols="50" name="message"></textarea>
 </div>
-<div class="logo5">
-</div>
+
+
 <div class="end_logo">
 <button type="submit" class="button_last">Get in Touch</button>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![mmm](https://user-images.githubusercontent.com/69638013/160253556-b189cfa9-ddd6-42a2-8012-e4081c770611.png) | ![nn](https://user-images.githubusercontent.com/69638013/160253548-6c09243d-d00b-4144-9ba4-90ff88362abf.png)

- [ ] Kindly center the Get in touch button, to look like the Figma design  as this was requested by the previous reviewer 👍🏿